src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -500,9 +513,9 @@ bool isSupportedConversion(Conversion conversion, BoundExpression operandOpt)
                 {
                     case ConversionKind.Identity:
                     case ConversionKind.DefaultOrNullLiteral:
-                        return true;
                     case ConversionKind.ImplicitReference:
-                        return operandOpt?.IsLiteralNullOrDefault() == true;
+                    case ConversionKind.ImplicitNullable:
+                        return true;
```

Comment:
Just to confirm, the way to get here (`ImplicitNullable` case) is to have that conversion in a tuple literal, correct?

Reply:
Yes, a tuple or a tuple literal.

---
In reply to: [250806060](https://github.com/dotnet/roslyn/pull/32729#discussion_r250806060) [](ancestors = 250806060)